ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -78,8 +78,8 @@ tests:
     - chain: openshift-qe-node-density-heavy
     workflow: openshift-qe-installer-aws
   timeout: 5h0m0s
-- always_run: false
-  as: data-path-9nodes
+- as: data-path-9nodes
+  cron: 0 9 * * 4
```

Comment:
lets maybe push this one a little earlier, maybe start at midnight?

Reply:
this starts 4 AM EST

Reply:
Since Roshni said Wed morning to noon EST, I kept this way